src/index.tsx

```diff
@@ -76,17 +76,20 @@ export class SignatureCanvas extends Component<SignatureCanvasProps> {
   }
 
   // return a trimmed copy of the canvas
-  getTrimmedCanvas = (): HTMLCanvasElement => {
+   getTrimmedCanvas = (): HTMLCanvasElement => {
     // copy the canvas
-    const canvas = this.getCanvas()
-    const copy = document.createElement('canvas')
-    copy.width = canvas.width
-    copy.height = canvas.height
+    const canvas = this.getCanvas();
+    const copy = document.createElement("canvas");
+    copy.width = canvas.width;
+    copy.height = canvas.height;
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    copy.getContext('2d')!.drawImage(canvas, 0, 0)
+    copy.getContext("2d")!.drawImage(canvas, 0, 0);
     // then trim it
-    return trimCanvas(copy)
-  }
+    // Handle both CJS and ESM default export forms
+    const trimFn = (trimCanvasImport as any).default || trimCanvasImport;
+
+    return typeof trimFn === "function" ? trimFn(copy) : copy;
```

Comment:
The function should not no-op if there is a problem, especially with no logging to the user that something went wrong. Such behavior would be very misleading to users as it clearly breaks the API contract